### DISCUSSION TITLE
Disable hotplug on ipxe

### DISCUSSION THREAD

**Discussion by CJohannimloh** [ipxe/ipxe#462] [Q&A] [answered]
Hello there!

We are using IPXE in combination with KVM-switches. 
When switching through the displays ipxe sometimes is getting stucked because the kvm-switch fails to emulate the graphic interface correctly.

**Is it possible to disable the hotplug feature in ipxe?**

BR
Christopher

**Comment by mcb30** [comment]
Could you describe the precise symptoms?  iPXE supports USB hotplug (if you have chosen to build with USB drivers included, which is unusual), but doesn't have any concept of graphical hotplug.

**Comment by CJohannimloh** [comment, marked as answer]
- We are booting an snponly.efi or undionly.efi using dnsmasq
   -  The script is chainloading a boot.ipxe from a IIS Webservice

When swichting through our displays ipxe sometimes freezes.
On a linux system we could just disable hotplug and the all devices are running without any issue.

But when IPXE doesn't support any graphical hotplug feature this isn't our problem.

**Reply by mcb30** [reply]
Thanks for the information.  There is no such thing as `undionly.efi`, so I presume you must mean `undionly.kpxe`.  The builds that you are using definitely do not include any USB drivers.

Any hotplug capability (for USB or display) will be provided by the platform firmware, and is outside the control of iPXE.  You should find that you get similar failures if you leave the system booted into another firmware-based application, such as the UEFI shell.

**Reply by CJohannimloh** [reply]
Yes indeed! It is undionly.kpxe -

Okay thank you very much! I'll try to narrow down our issue.
